validation/cubed_sphere_eddying_aquaplanet/cubed_sphere_eddying_aquaplanet.jl

```diff
@@ -0,0 +1,232 @@
+    τx(φ) = 0.1 * exp(-10 * (φ - π/3)^2) + 0.1 * exp(-10 * (φ + π/3)^2) - 0.2 * exp(-8φ^2) + 0.19 * exp(-20φ^2)
+
+    coriolis = HydrostaticSphericalCoriolis(scheme=VectorInvariantEnstrophyConserving())
+    R = grid.faces[1].radius
+    Ω = coriolis.rotation_rate
+
+    # Streamfunction integrated using Mathematica
+    # Scale by Ω^2 * R^2 * sin(φ) to get units of momentum flux (m²/s²) on the sphere.
+    # Minus sign because a negative flux drives currents in the positive x-direction.
+    # Hmmm, I think I messed up the magnitude but the resulting wind stress patterns should do the trick for now. It's a little too strong (stronger than the 0.1 N/m² we expect from τx).
+    # I'm further scaling it down by a factor of 0.1 so the wind stress is even weaker (for stability/debugging).
+    ψ̃(λ, φ) = - 0.1 * Ω^2 * R^2 * sin(φ) * (-0.02802495608 * erf(1.054092553 * (π - 3φ)) - 0.06266570687 * erf(2.828427125φ) + 0.03765160933 * erf(4.472135955φ) + 0.02802495608 * erf(1.054092553 * (π + 3φ)))
+
+    λ̃(λ) = (λ + 180)/ 360 * 2π
+    φ̃(φ) = φ / 180 * π
+    ψ(λ, φ) = ψ̃(λ̃(λ), φ̃(φ))
+
+    u_top_flux, v_top_flux, ψ₀ = diagnose_velocities_from_streamfunction(ψ, grid)
+
+    u_bcs, v_bcs = cubed_sphere_surface_momentum_flux_bcs(u_top_flux, v_top_flux, grid)
+
+    ## Model setup
+
+    model = HydrostaticFreeSurfaceModel(
+               architecture = CPU(),
+                       grid = grid,
+         momentum_advection = nothing,
+               free_surface = ExplicitFreeSurface(gravitational_acceleration=0.1),
+                   coriolis = coriolis,
+        boundary_conditions = (u=u_bcs, v=v_bcs),
+                    closure = HorizontallyCurvilinearAnisotropicDiffusivity(νh=1000),
+                    tracers = nothing,
+                   buoyancy = nothing
+    )
+
+    # Some random noise to get things going.
+    ε(λ, φ, z) = 1e-8 * randn()
+    Oceananigans.set!(model, u=ε, v=ε)
+
+    ## Simulation setup
+
+    Δt = 2minutes
+
+    g = model.free_surface.gravitational_acceleration
+    gravity_wave_speed = sqrt(g * H)
+    min_spacing = filter(!iszero, grid.faces[1].Δyᶠᶠᵃ) |> minimum
+    wave_propagation_time_scale = min_spacing / gravity_wave_speed
+    gravity_wave_cfl = Δt / wave_propagation_time_scale
+    @info "Gravity wave CFL = $gravity_wave_cfl"
+
+    cfl = CFL(Δt, accurate_cell_advection_timescale)
+
+    simulation = Simulation(model,
+                        Δt = Δt,
+                 stop_time = 7days,
+        iteration_interval = 20,
+                  progress = Progress(time_ns()),
+                parameters = (; cfl)
+    )
+
+    fields_to_check = (
+        u = model.velocities.u,
+        v = model.velocities.v,
+        η = model.free_surface.η
+    )
+
+    simulation.diagnostics[:state_checker] =
+        StateChecker(model, fields=fields_to_check, schedule=IterationInterval(20))
+
+    output_fields = merge(model.velocities, (η=model.free_surface.η,))
```

Comment:
Might be interesting to try outputting `VerticalVorticityField`:

```suggestion
    using Oceananigans.Models.HydrostaticFreeSurfaceModels: VerticalVorticityField
    ζ = VerticalVorticityField(model)
    output_fields = merge(model.velocities, (η=model.free_surface.η, ζ=ζ ))
```